awx/main/scheduler/__init__.py

```diff
@@ -2,3 +2,4 @@
 #
 
 from awx.main.scheduler.task_manager import TaskManager # noqa
+from awx.main.scheduler.kubernetes import PodManager # noqa
```

Comment:
To avoid tying the package's implementation to a specific path, you can probably use a relative import for these: `from .kubernetes import PodManager`